package.json

```diff
@@ -31,12 +31,15 @@
   },
   "homepage": "https://github.com/shian15810/eslint-plugin-typescript-enum#readme",
   "dependencies": {
-    "@typescript-eslint/experimental-utils": "^5.3.0"
+    "@typescript-eslint/utils": "^6.7.2"
```

Comment:
This is the main change, and what removes the nested problematic `tsutils` package from the dependency tree: https://github.com/typescript-eslint/typescript-eslint/issues/7155#issuecomment-1692384149

@shian15810 what do you think about this?

Reply:
Could you bump this to major version 8 of `@typescript-eslint/utils`? Its latest version is currently `8.18.0`.

Reply:
Done!

https://github.com/shian15810/eslint-plugin-typescript-enum/pull/338/commits/34496c58dd7c75bf6243333122da57784961b0fc